controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -90,6 +108,8 @@ def prepare(supervisor: Supervisor) -> None:
     for _, robot in get_robots(supervisor):
         robot.restartController()
 
+    supervisor.simulationSetMode(Supervisor.SIMULATION_MODE_REAL_TIME)
+
```

Comment:
This looks like a behaviour change -- is this intentional?

Reply:
This is related to ensuring that we do actually capture video, relates to https://github.com/cyberbotics/webots/issues/2386.

Reply:
This becomes moot once we have `wait_start` support from #205.